### PR TITLE
Add a playbook that prepares external VMs for usage with minishift

### PR DESCRIPTION
**Pull request by dirgim**
Add a playbook that prepares external VMs (fedora27+ or centos7) for usage with minishift including the option to setup nested virtualization on them.

### CODE REVIEW

File: playbooks/prepare_vm_for_minishift.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        state: present
+    - name: "Adding existing user {{ ansible_user }} to group docker"
+      user:
+        name: "{{ ansible_user }}"
+        groups: docker
+        append: yes
+    - name: "Ensure that docker service is running"
+      service:
+        name: docker
+        state: reloaded
```
Comment (skatlapa) [inline playbooks/prepare_vm_for_minishift.yml]: Idk if reload alone will start the service, what if it's not started before this task, it won't restart imo. Maybe we are better off using started or use a debug task? Except this, rest lgtm.

File: playbooks/prepare_vm_for_minishift.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+      raw: test -e /usr/bin/python || sudo yum install -y python
+    - name: "Install docker, net-tools and firewalld"
+      package:
+        name: "{{ item }}"
+        state: present
+      with_items:
+        - docker
+        - net-tools
+        - firewalld
+    - name: "Ensure group docker exists"
```
Comment (skatlapa) [inline playbooks/prepare_vm_for_minishift.yml]: nit: not really a stickler, but maybe space between task could be pleasant to the eye :) but we could do it next time if need be.

File: playbooks/prepare_vm_for_minishift.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+- hosts: '{{ hosts | default("all") }}'
+  become: '{{ become | default("yes") }}'
+  gather_facts: '{{ gather_facts | default("no") }}'
+  vars:
+    setup_nested_virt: false
+  tasks:
+    - name: "Install python if not installed"
+      raw: test -e /usr/bin/python || sudo yum install -y python
```
Comment (skatlapa) [inline playbooks/prepare_vm_for_minishift.yml]: nit: for dnf maybe use of another "||" if we don't want to use pip.

**Comment by skatlapa** [comment]
Except, reloaded vs started, rest LGTM.

File: playbooks/prepare_vm_for_minishift.yml (hunk 1 above)
Comment (dirgim) [reply playbooks/prepare_vm_for_minishift.yml]: The reloaded state starts the service if it's not already started.

File: playbooks/prepare_vm_for_minishift.yml (hunk 2 above)
Comment (dirgim) [reply playbooks/prepare_vm_for_minishift.yml]: Yep, it was a bit crowded. Fixed it now.

File: playbooks/prepare_vm_for_minishift.yml (hunk 1 above)
Comment (skatlapa) [reply playbooks/prepare_vm_for_minishift.yml]: then we good, +2

**Pull request merged**